### ISSUE TITLE
Improve usability of html-reporter

### ISSUE THREAD

**Issue by nonkor** [Keypr/helium#10]
I use html-report for debugging intensively and it definitely requires some improvements. So, my suggestions:

1. Instead of wrapping API call into plane lines like
```
---> HTTP POST https://<endpoint>
...
<--- END HTTP (64 body)
---> HTTP GET https://<endpoint>
...
<--- END HTTP (16 body)
```
which is really hard to parse due to a massive amount of such calls, I'd prefer to see something like:
```
HTTP POST https://<endpoint> (64 body)
  ...
HTTP GET https://<endpoint> (16 body)
  ...
```
where "..." is a body of request indented by 2 spaces. `HTTP POST` could be colored for visibility (e.g. "red").

2. Right now, `beforeEach` and `afterEach` hooks are not marked in a report at all. Considering descriptions of their actions are collapsed under the same header, it's hard to understand which calls there were performed **before** spec, which **after**. So, I'd prefer to see something like:
```
beforeEach {
  ...
}
afterEach {
  ...
}
```
where "..." are descriptions of hook requests indented by 2 spaces. `beforeEach {}` could be colored for visibility (e.g. "yellow").

**So, as a summary, I would welcome:**
- [ ] Indents for a body of api call instead of plain wrapping.
- [ ] Coloring.
- [ ] Marking hooks.

**Comment by Burbaka** [comment]
Definitely should be done.
I can add that from my side the following syntax enhancements should be done:
- increase the size of the text
- much bigger tabulation
- space between separate requests
